### ISSUE TITLE
Important

### ISSUE THREAD

**Issue by farishkash** [JakeNas/prj-rev-bwfs-tea-cozy#1]
https://github.com/JakeNas/prj-rev-bwfs-tea-cozy/blob/aee002063492790ea2514c033adaca2d1599209e/teacozy/index.html#L41

You should be using descriptive alt text and not this.

The reason being is the visually impaired uses screen readers.  The screen readers will read this to describe the image.  They may think your website is broken.
